Lib/test/test_ordered_dict.py

```diff
@@ -650,6 +650,41 @@ def test_free_after_iterating(self):
         support.check_free_after_iterating(self, lambda d: iter(d.values()), self.OrderedDict)
         support.check_free_after_iterating(self, lambda d: iter(d.items()), self.OrderedDict)
 
+    # TODO: This test is relating to PEP 468.
+    # Move this test to somewhere more appropriate.
+    def test_kwargs_order(self):
```

Comment:
This test is about `kwargs` ordering rather than `OrderedDict`, so I'd expect it to be next to other tests for `kwargs`, not here in `test_ordered_dict.py`.

Reply:
I added new test class for this, in test_call.py